### ISSUE TITLE
Details Row used in Grouped List (not Details Grouped List) doesn't announce selected state

### ISSUE THREAD

**Issue by bcoard** [microsoft/fluentui#16470]
### Environment Information

- **Package version(s)**: ^7.155.1
- **Browser and OS versions**: Chromium Edge; OS Version 1903/18362.10000

### Describe the issue:

Details Row items used within a Grouped List doesn't announce whether it's selected or unselected via Narrator. As far as I can tell all properties are set but Maybe I'm just looking at the wrong control. From a user standpoint, while navigating through the list with Narrator enabled, the user should be able to tell which list items are selected and which ones aren't.

The Group List document page could be used as a repro: https://developer.microsoft.com/en-us/fluentui#/controls/web/groupedlist

### Please provide a reproduction of the issue in a codepen:

1. Turn Narrator on.
2. Using keyboard commands navigate to a Details Row list item (in a Grouped List).

#### Actual behavior:

Narrator doesn't announce selected state.

#### Expected behavior:

Narrator should announce whether the list item is selected or unselected.

**Comment by gouttierre** [comment]
@bcoard - thanks for your input. I followed your steps you mentioned in the issue and was unable to repo the problem stated. Can you confirm if you are still experiencing the problem? Thanks

**Comment by smhigley** [comment]
@bcoard are you able to reproduce on your computer on the latest v8 (https://fabricweb.z5.web.core.windows.net/oufr/8.1.8/#/examples/groupedlist)? For me, Narrator reads selected/not selected state both when navigating to rows and when toggling row selection:

https://msit.microsoftstream.com/video/b92aa1ff-0400-85a8-5a9b-f1eb81d1bad8

(apologies to anyone outside Microsoft who can't view that link 😅. It just shows Narrator reading the row text, then "selected" or "not selected", and reading "space, selected" when toggling selection with the space key).

That was Edge v89, and my Windows build is 19042

**Comment by bcoard** [comment]
Good thought. Looks like it still repros for me in v8.

Edge v90.0.803.0
Windows 19043.844

**Comment by bcoard** [comment]
I believe @smhigley actually fixed this with a recent PR. I'll let her comment and report back or open a new bug if this pops up again.

**Comment by smhigley** [comment]
Confirmed this works for me with Narrator + Edge Chromium in v8, so closing this issue as fixed (I honestly don't remember which detailslist/groupedlist change was related to this issue anymore 😂)
